content/departments/product-engineering/engineering/cloud/delivery/deployment/helm.md

```diff
@@ -48,3 +48,98 @@ In the event you need assistance from Delivery, please gather the following info
+
+#### Can you output the manifest files from the Helm chart?
+The helm template command can be used to render Kubernetes manifests from the chart. This output can then be inspected, checked into source control, or applied to the cluster via kubectl.
+
+More about this in the [Reviewing changes](https://docs.sourcegraph.com/admin/install/kubernetes/helm#reviewing-changes) section of our Helm docs.
+
+#### What are the limitations of using Helm?
+Without opting to either use Kustomize on top of Helm, or create a sub chart ([read more about both here](https://docs.sourcegraph.com/admin/install/kubernetes/helm#advanced-configuration)), customers are limited to the configuration options built into the chart by Sourcegraph. 
+
+If a customer needs additional features not included in the chart, it needs to be requested to be added.
+
+#### Do we support converting Docker Compose/Kustomize/plain Kubernetes to use Helm?
+Not yet, but it’s planned for FY23-Q2/3.
+
+Instead, it should be used on a new installation if a customer really wished to move to Helm. 
+
+If a customer wishes to try and convert themselves, any customizations will need to be converted to a override file ([see here for more info](https://docs.sourcegraph.com/admin/install/kubernetes/helm#configuration)). The [sourcegraph/examples folder](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-helm/-/tree/charts/sourcegraph/examples) in the deploy-sourcegraph-helm repo has some samples of commonly modified values that can be used as a starting point.
+
+#### Can a customer who’s written their own Helm chart migrate to ours?
+In theory, yes, and if they have the knowledge to write their own, migrating to ours should be simple enough. But the Delivery team are happy to consult on this process, with reasonable notice!
+
+#### Will we be deprecating Kustomize/Docker Compose?
+There are no current plans to deprecate either deployment option. If that changes, there will be a generous deprecation timeline and support for the migration.
+
+#### Have any of the Sourcegraph deployment defaults changed as part of the move to Helm?
+Yes - with the helm deployment, the Sourcegraph cluster defaults to running as non-root users with a read-only root filesystem. This is similar to running with the non-privileged kustomize overlay. This default was changed to provide a smoother setup experience in Kubernetes clusters with restricted access and to follow best practices for security. The security context for each deployment can be customized to match any unique requirements for your cluster - see [this example](https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph-helm/-/tree/charts/sourcegraph/examples/custom-security-context) in the deploy-sourcegraph-helm repo.
+
+#### What additional customization options are planned for the future? 
+That’s very dependent on customer requests, but the ones we have in mind include:
+- TODO
```

Comment:
@michaellzc @caugustus-sourcegraph anything to add here??

Reply:
I don't have anything specific on top of my head. Maybe just leave it as it is.

Reply:
I took a peek at our open tickets and it doesn't look like we have anything lined up for post-GA yet. The only thing that comes to mind is general improvements to the usability of the chart - i.e., automatically setting environment variables based on different settings. For example, automating the configuration in https://docs.sourcegraph.com/admin/external_services/object_storage.

Customer feedback here is going to direct most of our efforts though.